include/swift/Parse/Parser.h

```diff
@@ -38,7 +38,7 @@
 #include "llvm/ADT/SetVector.h"
 
 namespace llvm {
-  template <typename PT1, typename PT2, typename PT3> class PointerUnion3;
+  template <typename PT1, typename PT2, typename PT3> class PointerUnion;
```

Comment:
This forward declaration isn't correct anymore, is it?

Reply:
Yeah, this should be:
```
template <typename ...T> class PointerUnion;
```